client/my-sites/stats/activity-log/threat-alert.jsx

```diff
@@ -0,0 +1,61 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component, Fragment } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import ActivityIcon from '../activity-log-item/activity-icon';
+import FoldableCard from 'components/foldable-card';
+
+export class ThreatAlert extends Component {
+	state = {
+		isExpanded: false,
```

Comment:
This doesn't seem to be used.

Reply:
thanks! good catch - I was using it before using the `<FoldableCard />`

Reply:
removed in 2ba4279571